### ISSUE TITLE
[pyclustering.container.cftree] Lost CF-nodes during inserting

### ISSUE THREAD

**Issue by annoviko** [annoviko/pyclustering#304]
Test scenario: traverse whole tree and count nodes and then compare with CF-tree statistical value.
```python
    def templateLevelNodeObtaining(self, number_points, branching_factor):
        tree = cftree(branching_factor, 1, 0.1);
         
        for index_point in range(0, number_points):
            point = [ index_point ];
            tree.insert_cluster([ point ]);
        
        total_node_amount = 0;
        for level in range(0, tree.height):
            nodes = tree.get_level_nodes(level);
            total_node_amount += len(nodes);
        
        assert tree.amount_nodes == total_node_amount;

    def testLevelNodeObtaining_7_2(self):
        self.templateLevelNodeObtaining(7, 2);

    def testLevelNodeObtaining_10_2(self):
        self.templateLevelNodeObtaining(10, 2);

    def testLevelNodeObtaining_7_3(self):
        self.templateLevelNodeObtaining(7, 3);

    def testLevelNodeObtaining_20_3(self):
        self.templateLevelNodeObtaining(20, 3);

    def testLevelNodeObtaining_26_3(self):
        self.templateLevelNodeObtaining(26, 3);

    def testLevelNodeObtaining_16_4(self):
        self.templateLevelNodeObtaining(16, 4);

    def testLevelNodeObtaining_34_4(self):
        self.templateLevelNodeObtaining(34, 4);
```

Thus inserting procedure is not accurate and lead to unbalanced tree.

**Comment by annoviko** [comment]
Fix propagation to 0.6.x is required.

**Comment by annoviko** [comment]
Correction for 0.6.x: https://github.com/annoviko/pyclustering/commit/c705327565094a8ab9026bcd8045948bd5edc186 . The issue has been resolved.
